fiftyone/utils/paddleocr.py

```
@@ -0,0 +1,418 @@
+        pass  # paddleocr downloads the model on first load
+
+    def _load_model(self, config):
+        return paddleocr.TextDetection(
+            model_name=config.det_model_name, engine="transformers"
+        )
+
+    @property
+    def media_type(self):
+        return "image"
+
+    def _forward_pass(self, imgs):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_forward_pass`

(ANN202)